module/Application/Module.php

```diff
@@ -32,6 +32,7 @@ public function onBootstrap(MvcEvent $e)
         $translator->setlocale($locale);
 
         Carbon::setLocale($locale);
+        \Locale::setDefault($locale);
```

Comment:
Will setting the locale globally break anything? I need to set it here so that the meeting date is displayed correctly.

Reply:
Probably not. But maybe this should be rolled out on test first and tested for a bit. Especially looking for dates in various areas (such as with photos and activities).

Reply:
Has this been tested already?

Reply:
No, not yet. I still have to look into it. My intuition also says that line 34 might be unnecssary because Carbon uses the global locale. I'll have to check first.

Reply:
I did some investigation and I've concluded that (1) setting the locale globally will not break anything, and (2) setting the Caron locale separately is needed.

Regarding (1), I'm using `\Locale::setDefault()` so that the view helper `dateFormat()` correctly outputs formats the date on the meeting page ([line 9](https://github.com/GEWIS/gewisweb/pull/965/files#diff-4ee0de3a261ae71a9f007aae25b88f8b)).

The `dateFormat()` view helper is used 18 times throughout our code base. Every call to this view helper passes a fourth parameter specifying the session's locale ([example](https://github.com/GEWIS/gewisweb/blob/a2216e34885737f28f25ae805501a1c206816d8b/module/Activity/view/email/activity.phtml#L15)). In fact, I came across a couple of occurrences where the locale was hardcoded ([example](https://github.com/GEWIS/gewisweb/blob/a2216e34885737f28f25ae805501a1c206816d8b/module/Decision/view/decision/member/birthdays.phtml#L26-L30)). So, setting the Locale globally will not affect these calls as the fourth parameter overwrites the global default. Furthermore, Locale only affects functions that use the value returned by `Locale::getDefault()` (i.e. has no system side effects). Zend calls `getDefault()` a couple of times internally, but it should be fine.

Regarding (2), it tried removing line 34 and Carbon defaulted to English, ignoring the locale.

Reply:
If you agree, you can resolve this conversation and proceed to merge this PR.

Reply:
Looks good. Nice work!